frontend-react/src/pages/admin/AdminReceiverDashPage.tsx

```diff
@@ -0,0 +1,34 @@
+import { NetworkErrorBoundary } from "rest-hooks";
+import { Helmet } from "react-helmet";
+import { Suspense } from "react";
+
+import { ErrorPage } from "../error/ErrorPage";
+import Spinner from "../../components/Spinner";
+import HipaaNotice from "../../components/HipaaNotice";
+import { AdminReceiverDashboard } from "../../components/Admin/AdminReceiverDashboard";
+
+export function AdminReceiverDashPage() {
+    return (
+        <NetworkErrorBoundary
```

Comment:
@kevinhaube's thoughts on this.

It would be best if network errors were handled in component that generated them, but the way coinbase's RestHooks works is that the error handling happens OUTSIDE the component.

Reply:
The short of it is NetworkErrorBoundary is from `rest-hooks` but picks up any network error. Think of it like a catch block for components. It'll catch whatever throws, and render whatever the ErrorBoundary component tells it to render. We could (and likely will) make our own and show different things based on error codes. Suspense is the equivalent of `if (loading) <Spinner />` as a higher-order component for _everything_ it wraps...so if any of those components wrapped in it fetch anything, until the last one loads its data, we'll have a spinner.

There's some helpful info in our Tech-Debt ticket about this!

Reply:
Yeah, I just followed the format of the outer page I saw in other files and didn't give much thought to it. I'll investigate what the right answer is.

Reply:
I removed the outer `<NetworkErrorBoundary>` in the latest update